libsolidity/codegen/YulUtilFunctions.cpp

```diff
@@ -3765,7 +3765,7 @@ string YulUtilFunctions::negateNumberWrappingFunction(Type const& _type)
 	return m_functionCollector.createFunction(functionName, [&]() {
 		return Whiskers(R"(
 			function <functionName>(value) -> ret {
-				value := <cleanupFunction>(sub(0, value)))
+				value := <cleanupFunction>(sub(0, value))
 			}
```

Comment:
Anyone wants to investigate why this template was terminated? This trailing `}` was not inserted in the output.

Reply:
Probably just because the output is only after reparsing, which stopped parsing at the invalid ``)``?

Reply:
No, it did not stop. The main issue reported was missing `}`.

Reply:
See https://app.circleci.com/pipelines/github/ethereum/solidity/12409/workflows/45d8ff35-54a1-4ee0-8b0f-2395a634da51/jobs/583227/tests#failed-test-3

<img width="1091" alt="Screenshot 2021-01-13 at 20 42 41" src="https://user-images.githubusercontent.com/20340/104508267-e308ba00-55df-11eb-8864-d2aa6ba86bca.png">

Reply:
Oh wow, just realised it is `value := ` where it should be `ret :=`.

Reply:
Wow